[2018-10-16 13:47:11] my-log.ERROR: I did it! [] []
